### PR TITLE
Added Ezoa.ECHOOF

### PR DESCRIPTION
**Pull request by ezoa-page**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/172115)

### CODE REVIEW

**Comment by ezoa-page** [comment]
@microsoft-github-policy-service agree

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please move to multi-file manifest format.

File: manifests/e/Ezoa/ECHOOF/0.1.1/Ezoa.ECHOOF.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PackageIdentifier: "Ezoa.ECHOOF"
+PackageVersion: "0.1.1"
+PackageLocale: "en-US"
+Publisher: "Ezoa"
+PackageName: "{{name}"
```
Comment (stephengillie) [inline manifests/e/Ezoa/ECHOOF/0.1.1/Ezoa.ECHOOF.locale.en-US.yaml:5]:
```suggestion
PackageName: "ECHOOF"
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Verify:
image

Hi @ezoa-page,

I'm not entirely sure, but I believe `PackageName` has to reflect the Registry entry.

File: manifests/e/Ezoa/ECHOOF/0.1.1/Ezoa.ECHOOF.locale.en-US.yaml (hunk 1 above)
Comment (ezoa-page) [reply manifests/e/Ezoa/ECHOOF/0.1.1/Ezoa.ECHOOF.locale.en-US.yaml:5]: Ah that's embarassing, I missed a character in my manifest generator. Thank you for pointing that out!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**